composite/market-closure/src/index.ts

```diff
@@ -1,6 +1,8 @@
 import { expose } from '@chainlink/ea-bootstrap'
-import { execute } from './adapter'
+import { makeExecute } from './adapter'
+import { makeConfig } from './config'
 
 const NAME = 'MarketClosure'
```

Comment:
Could we make this something like `MARKET_CLOSURE`? Or `MARKET-CLOSURE`.. or `MARKETCLOSURE`. Not sure what pattern we want but we should be consistent.